cmd/agent/subcommands/run/dependent_services_windows.go

```diff
@@ -22,43 +23,47 @@ type serviceInitFunc func() (err error)
 // Servicedef defines a service
 type Servicedef struct {
 	name       string
-	configKeys []string
+	configKeys map[config.Config]string
 
 	serviceName string
 	serviceInit serviceInitFunc
 }
 
 var subservices = []Servicedef{
 	{
-		name:        "apm",
-		configKeys:  []string{"apm_config.enabled"},
+		name: "apm",
+		configKeys: map[config.Config]string{
+			config.Datadog: "apm_config.enabled",
+		},
 		serviceName: "datadog-trace-agent",
 		serviceInit: apmInit,
 	},
 	{
 		name: "process",
-		configKeys: []string{
-			"process_config.enabled",
-			"process_config.process_collection.enabled",
-			"process_config.container_collection.enabled",
-			"process_config.process_discovery.enabled",
-			"network_config.enabled",
-			"system_probe_config.enabled",
+		configKeys: map[config.Config]string{
+			config.Datadog:     "process_config.enabled",
```

Comment:
I might be missing some go lore, but won't this map only wind up with two entries ? you might want to reverse the keys and values

https://play.golang.com/p/zeaxiOxX9kQ

Reply:
Derp, you are right.